tck/src/main/java/org/eclipse/microprofile/openapi/tck/ModelConstructionTest.java

```diff
@@ -862,7 +867,13 @@ public void securityRequirementTest() {
         
         final List<String> otherValue = new ArrayList<String>();
         checkMapImmutable(sr, SecurityRequirement::getSchemes, "otherScheme", otherValue);
-        checkNullValueInAdd(sr::getSchemes, sr::addScheme, "other", otherValue);
+        
+        final String schemeKey3 = "myScheme3";
+        sr.addScheme(schemeKey3, (String) null);
+        assertTrue(sr.hasScheme(schemeKey3), "Expected " + schemeKey3 + " to be present");
+        final List<String> schemeValue3 = Collections.emptyList();
+        assertSame(sr.getScheme(schemeKey3), schemeValue3, 
```

Comment:
For list comparison, please use `assertEquals(..)`, checking for the same instance is too strong (implementation can use something else than `Collections.emptyList()`)

```suggestion
        assertEquals(sr.getScheme(schemeKey3), schemeValue3, 
```